lib/philomena/users/user.ex

```diff
@@ -347,7 +347,10 @@ defmodule Philomena.Users.User do
       :show_sidebar_and_watched_images
     ])
     |> TagList.propagate_tag_list(:watched_tag_list, :watched_tag_ids)
-    |> validate_inclusion(:theme, ~W(default dark red))
+    |> validate_inclusion(
+      :theme,
+      ~W(dark-red dark-orange dark-yellow dark-blue dark-green dark-purple dark-cyan dark-pink dark-silver light-red light-orange light-yellow light-blue light-green light-purple light-cyan light-pink light-silver)
```

Comment:
It would be more concise to write it like this
```elixir
defmodule Philomena.Users.User
  # ...
  def theme_colors do
    ~W(red orange yellow blue green purple cyan pink silver)
  end

  def theme_names do
    ~W(dark light)
  end

  def themes do
    for name <- theme_names(), color <- theme_colors() do
      "#{name}-#{color}"
    end
  end

  # ...
  |> validate_inclusion(:theme, themes())
```